homeassistant/exceptions.py

```diff
@@ -13,6 +13,9 @@
 class HomeAssistantError(Exception):
     """General Home Assistant exception occurred."""
 
+    _message: str | None = None
+    _translate: bool = False
```

Comment:
Maybe name this `generate_message` or something to make it clear we don't translate to the configured language.